README.md

```diff
@@ -110,3 +113,23 @@ Open a PR against [maintainers.md](maintainers.md) adding your email and Github
 For your information, the 'owner' of elm-community (in Github terms) is
 @rgrempel. So, feel free to contact me if there are things that need to be
 done that require an 'owner'.
+
+
+## Documentation for Maintainers
+
+### Renaming Elm-Community Packages
+
+User packages tend to be prefixed like `elm-html-extra`. When a package is
+transfered to the elm-community team, the `elm-` prefix becomes
+redundant(`elm-community/elm-html-extra`), so the prefix may be removed from
+the original package's name:
+
+* Name changes should only occur when a package is upgraded to a newer Elm
```

Comment:
@prikhi Make it more explicit for when a repo is _moved_ too - not just when a new package is created